Khan/perseus
# [Locked Figure Labels] Add/edit/delete locked ellipse labels

## Summary:
Add the ability to add/edit/delete labels for locked ellipses
within the interactive graph editor.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2349

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-ellipse-labels-flag
- Go to locked figures
- Open the locked ellipse settings
- Confirm that the label settings are there
- Play around with the label settings and confirm that it moves
  and updates with the ellipse

<img width="320" alt="image" src="https://github.com/user-attachments/assets/55bf06c9-1254-4bc3-8ef1-f57bce7e2028">
